SpringProject/pom.xml

```diff
@@ -24,6 +26,25 @@
 			<artifactId>spring-boot-starter</artifactId>
 		</dependency>
 
+<!--		<dependency>-->
+<!--			<groupId>org.flywaydb</groupId>-->
+<!--			<artifactId>flyway-core</artifactId>-->
+<!--		</dependency>-->
```

Comment:
Remove this dependancy if it is not being used

Reply:
In our version this is not commented out, do you think we need to remove it?